models/articles.modles.js

```diff
@@ -1,5 +1,34 @@
 const db = require('../db/connection');
 
+exports.selectArticles = () => {
+  const queryStr = `SELECT 
+  articles.author, 
+  articles.title, 
+  articles.article_id, 
+  articles.topic, 
+  articles.created_at, 
+  articles.votes, 
+  articles.article_img_url,
+  COALESCE(COUNT(comments.comment_id), 0)::int as comments_count
```

Comment:
I'm not sure the `COALESCE` is necessary here, if you instead count `comments.article_id` - maybe give it a try

Reply:
Fixed, not sure how it works